templates/api/ingress.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if and .Values.api.enabled .Values.api.ingress.enabled -}}
 {{- $fullName := include "orchestrate.api.fullname" . -}}
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
Available since 1.19, which is relatively new. The default ingress helm template has a pretty good coverage.

Could we support all the ingress API versions?

Reply:
This was intentional, as the previous versions could not be tested we have chosen not to go below 1.19